app/src/androidTest/java/org/squidfish/tuplenback/LocalStorageRepositoryTests.kt

```diff
@@ -0,0 +1,85 @@
+package org.squidfish.tuplenback
+
+import android.content.Context
+import androidx.room.Room
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.test.runTest
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.squidfish.tuplenback.data.LocalStorageRepository
+import org.squidfish.tuplenback.data.room.AppDatabase
+import org.squidfish.tuplenback.games.Game
+import org.squidfish.tuplenback.games.GameModule
+import org.squidfish.tuplenback.models.GameStatsModel
+
+@RunWith(AndroidJUnit4::class)
```

Comment:
This is automatically the annotation for tests in the `androidTests` folder.
These tests look simple enough for us to pull on the JVM. Take a look at Robolectric